OpenDreamClient/EntryPoint.cs

```diff
@@ -18,6 +19,14 @@ public sealed class EntryPoint : GameClient {
         [Dependency]
         private readonly IDreamResourceManager _dreamResource = default!;
 
+        private const string UserAgent =
+            "Mozilla/4.0 (compatible; MSIE 7.0; Windows NT 6.2; WOW64; Trident/7.0; .NET4.0C; .NET4.0E; .NET CLR 2.0.50727; .NET CLR 3.0.30729; .NET CLR 3.5.30729; Tablet PC 2.0; Zoom 3.6.0) ";
```

Comment:
```suggestion
            "Mozilla/4.0 (compatible; MSIE 7.0; Windows NT 6.2; WOW64; Trident/7.0; .NET4.0C; .NET4.0E; .NET CLR 2.0.50727; .NET CLR 3.0.30729; .NET CLR 3.5.30729) ";
```
I think we can safely not spoof being a tablet or having Zoom installed.

Reply:
Fixed in [`97e6ce9` (#1129)](https://github.com/OpenDreamProject/OpenDream/pull/1129/commits/97e6ce939abca0949b25ea6e7c64fa6c99c48bd4)